starknet_py/net/account/account.py

```diff
@@ -71,6 +70,9 @@ def __init__(
         chain: Optional[StarknetChainId] = None,
     ):
         """
+        Creates an Account instance.
+        If only `signer` is given, it will be used in Account.
+        If only `key_pair` is given, `client.net` will be used to create `chain`.
```

Comment:
That is quite strange. Although I know what is the purpose of that PR I'm not sure if I understand that 😅